src/electron/common/content-page-info-factory.ts

```diff
@@ -0,0 +1,21 @@
+import { ContentPageInfo, ContentPagesInfo } from 'electron/types/content-page-info';
+import { LeftNavItemKey } from 'electron/types/left-nav-item-key';
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { TestConfig } from 'electron/types/test-config';
+
+export const createContentPagesInfo = (configs: TestConfig[]): ContentPagesInfo => {
+    const retVal = {} as ContentPagesInfo;
```

Comment:
Since it's an empty object, it doesn't need the "as ContentPagesInfo. You should be able to use normal definition like:

`retVal: ContentPagesInfo`

Reply:
If this isn't there, we get an eslint error saying that other standard object properties can't be found.

Reply:
Hm very odd.